package.json

```
@@ -132,10 +132,10 @@
     "sourcemap-codec": "^1.4.8",
     "systemjs": "0.18.10",
     "terser": "^4.4.0",
-    "tsickle": "0.38.0",
+    "tsickle": "^0.38.1",
```

```suggestion
    "tsickle": "0.38.1",
```
we don't want to update tsickle by accident/unintentionally.